src/main/webapp/MarkerTemplate.js

```diff
@@ -12,19 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Defines the basic functions that all markers must have
+ * All classes that extend MarkerTemplate must implement:
+ *  - openMyInfoWindow()
```

Comment:
In JsDoc, this is done by defining empty methods and annotating them with `@abstract`.
Ex:
```
/**
 * Opens info window.
 *
 * @abstract
 */
openMyInfoWindow() {}
```

Reply:
Duly noted!